integration/README.md

```diff
@@ -0,0 +1,48 @@
+# Raiden Integration Image
+
+## About
+
+This integration image acts as a base to ran integration tests for the Light Client in a fully controlled, 
+contained environment.
+
+The environment contains:
+- Geth running a private chain using the Clique PoA engine.
+- [Raiden Contracts](https://github.com/raiden-network/raiden-contracts) pre deployed
+- Matrix Synapse
+- Pathfinding Service ([raiden-services](https://github.com/raiden-network/raiden-services))
+- A `CustomToken` deployed with a `Token Network` deployed
+- Two [Raiden](https://github.com/raiden-network/raiden) nodes with a pre funded open channel
+
+## Using the image
+
+### Building
+
+To build the docker image you need to run:
+
+```bash
+docker build -t lightclient-integration .
+```  
+  
+### Running
+
+To start the container you can run:
+
+```bash
+docker run --name lc-integration -p 80:80 -p 6000:6000 -p 5001:5001 -p 5002:5002 -p 8545:8545 lightclient-integration
```

Comment:
May I recommend prepending `localhost:` to the ports mapping, so they aren't exposed outside if the user doesn't have a proper firewall configured?